db/migrate/20220613123730_add_export_fields.rb

```diff
@@ -0,0 +1,13 @@
+class AddExportFields < ActiveRecord::Migration[7.0]
+  def change
+    change_table :case_logs, bulk: true do |t|
+      t.column :hhtype, :integer
+      t.column :new_old, :integer
+      t.column :vacdays, :integer
+      t.rename :tenant_code, :tenancycode
+      t.rename :previous_postcode_known, :ppcodenk
```

Comment:
Possible this triggers a couple of seconds of downtime/Sentry crash if one of the servers still has old application code after the first server has finished running migrations but should be fine.